### DISCUSSION TITLE
Why transition base image from nixos/nix to debian?

### DISCUSSION THREAD

**Discussion by Pegasust** [railwayapp/nixpacks#469] [Q&A] [answered]
I learned about Nixpacks recently and just today I wondered what would be the pros and cons of building an image entirely on NixOS vs another distro + Nix pkm.

I came across a point back in April when nixos/nix was still used.

Is there compatibility or performance reasons that makes the transition sound?

**Comment by Milo123459** [comment, marked as answer]
We have options to install APT packages as well as Nix packages, so we decided to use the debian base image.
